packages/webapp/pages/account/profile.tsx

```diff
@@ -255,6 +262,17 @@ const AccountProfilePage = (): ReactElement => {
           valid={!hint.github}
           name="github"
           value={user?.github}
+          placeholder="username or url"
+        />
+        <AccountTextField
+          leftIcon={<LinkedInIcon />}
+          label="LinkedIn"
+          inputId="linkedin"
+          hint={hint.linkedin}
+          valid={!hint.linkedin}
+          name="linkedin"
+          value={user?.linkedin}
+          placeholder="username or url"
```

Comment:
```suggestion
          placeholder="Username or URL"
```
😬

Reply:
On the others too, that are not a url

Reply:
Applied in 9edb90c5222e766f0b0d0642e62a413f98770e7b